misc/xml-to-record-converter/src/main/java/io/ballerina/xmltorecordconverter/XMLToRecordConverter.java

```diff
@@ -91,8 +93,9 @@ private XMLToRecordConverter() {}
     private static final String XMLDATA = "xmldata";
     private static final String COLON = ":";
 
-    public static XMLToRecordResponse convert(String xmlValue, boolean isRecordTypeDesc, boolean isClosed,
-                                              boolean forceFormatRecordFields) {
+    public static XMLToRecordResponse convertXMLToRecord(String xmlValue, boolean isRecordTypeDesc, boolean isClosed,
+                                                         boolean forceFormatRecordFields, String value,
```

Comment:
Here I didn't get the meaning of value, shall we have a self-explanatory argument name?